src/main/java/org/littil/api/user/api/UserResource.java

```diff
@@ -84,6 +88,37 @@ public Response get(@Parameter(name = "id", required = true) @PathParam("id") fi
         return Response.status(Response.Status.NOT_FOUND).build();
     }
 
+    @GET
+    @Path("user/provider/{id}")
+    @Authenticated
+    @Operation(summary = "Fetch a specific user by provider Id")
+    @APIResponse(
+            responseCode = "200",
+            description = "User with provider Id found.",
+            content = @Content(
+                    mediaType = MediaType.APPLICATION_JSON,
+                    schema = @Schema(type = SchemaType.OBJECT, implementation = User.class)
+            )
+    )
+    @APIResponse(
+            responseCode = "404",
+            description = "User with specific provider Id was not found."
+    )
+    @APIResponse(
+            responseCode = "403",
+            description = "The provider Id is not correct."
```

Comment:
This is a forbidden code. The description does not really match the code. Which one is leading?